pkg/bundle/manager_test.go

```diff
@@ -128,6 +128,60 @@ func TestBundleManager_ProcessBundleController(t *testing.T) {
 		rc.EXPECT().LatestBundle(ctx, testBundleRegistry+"/eks-anywhere-packages-bundles", testKubernetesVersion).Return(latestBundle, nil)
 		bc.EXPECT().GetBundleList(ctx).Return(allBundles, nil)
 		bc.EXPECT().CreateClusterNamespace(ctx, pbc.Name).Return(nil)
+		bc.EXPECT().GetBundle(ctx, pbc.Spec.ActiveBundle).Return(&allBundles[0], nil)
+
+		err := bm.ProcessBundleController(ctx, pbc)
+
+		assert.NoError(t, err)
+		assert.Equal(t, api.BundleControllerStateActive, pbc.Status.State)
+	})
+
+	t.Run("active missing active bundle", func(t *testing.T) {
```

Comment:
As mentioned above, I don't see a test that exercises the path when the active bundle is "".

Reply:
moved the "" activeBundle to manager

Reply:
These tests aren't for activeBundle=="", but for just a missing active bundle